python/coding-agent/cli.py

```
@@ -0,0 +1,91 @@
+                name="Chat Turn",
+                input_text=message,
+                session_id=session_id,
+                tags={"session_id": session_id},
+            )
+
+            # Show a live timer while the agent works
+            stop_spinner = threading.Event()
+
+            def spinner():
+                start = time.time()
+                while not stop_spinner.is_set():
```

Ruff (0.15.4)

[warning] Function definition does not bind loop variable `stop_spinner`

(B023)